source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,398 @@
+For Super Rentals, we want to be able to display all rentals and allow the user to view additional information for each.
+To do this, we can generate `rentals` route to display all rentals and a `show` sub-route to display additional information.
+
+# A Parent Route
+
+To begin, let's generate our new route.
+We'll call this `rentals`, since it will list all of our current rental properties.
+
+```shell
+ember g route rentals
+```
+
+You will see output like this:
+
+```text
+installing route
+  create app/routes/rentals.js
+  create app/templates/rentals.hbs
+updating router
+  add route rentals
+installing route-test
+  create tests/unit/routes/rentals-test.js
+```
+
+If it is unclear what Ember is doing for us here visit the [Routes and Templates tutorial](../routes-and-templates) for more information.
+
+Let's start by opening our new Handlebars template (`app/templates/rentals.hbs`).
+Ember generates route-level templates files with an `outlet`.
+
+```app/templates/rentals.hbs
+{{outlet}}
+```
+
+Much like our application template (`app/templates/application.hbs`), the `outlet` is where our sub-routes will render.
+This means that any content on our parent route will be present as we browse down through our child routes, allowing us to add things like navigation, footers or sidebars.
+Let's start by adding welcome header and link to our About page (`app/templates/about.hbs`).
+
+```app/templates/rentals.hbs
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Welcome!</h2>
+  <p>
+    We hope you find exactly what you're looking for in a place to stay.
+  </p>
+  {{#link-to 'about' class="button"}}
+    About Us
+  {{/link-to}}
+</div>
+
+{{outlet}}
+```
+
+Run `ember serve` (or `ember s` for short) from the shell to start the Ember development server,
+and then go to `localhost:4200/rentals` to see our new app in action!
+
+# Creating a Sub-route
+
+Next, we will want to create a sub-route that will list information for a specific rental.
+To do this, we will need to update a couple of files.
+To find a specific rental, we will want to use Ember Data's `queryRecord` function [Finding Records section](../../models/finding-records/).
+This means we need to be able to search by a unique key in order to return the specific rental we want.
+Many APIs return a `slug`, which is a URL-friendly string to identify a record.
+
+While on the `show` route, we will also want to show additional information about our specific rental.
+
+In order to do this, we need to modify the Mirage `config.js` file.
```

Comment:
I agree that I would like an acceptance test here because of all the moving parts.  Based on the comment here: https://github.com/emberjs/super-rentals/pull/24#discussion_r65718605, it sounded like an acceptance test tutorial would be separate and out of the scope here.  Happy to add whatever makes the most sense.